### PR TITLE
[P4-2663] Add handover date under print icon

### PR DESCRIPTION
**Pull request by merlinc**
## Proposed changes

Add an extra handover date/time underneath the print icon, as an affordance. Don't display this extra data when printed.

### Issue tracking

- [P4-2663](https://dsdmoj.atlassian.net/browse/P4-2663)

## Screenshots

Screenshot_2021-02-12 Move for AUGICK, 'NJINS – Book a secure move
Screenshot_2021-02-12 Person Escort Record for AUGICK, #39;NJINS – Book a secure move(2)

#### Automated testing

- [ ] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [ ] No environment variables were added or changed

- [ ] Documented in the [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md)
- [ ] Added to [continous integration](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-book-secure-move-frontend)
- [ ] Added to staging environment (deployment repository)
- [ ] Added to preproduction environment (deployment repository)
- [ ] Added to production environment (deployment repository)

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [ ] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics
- [ ] Update [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md) with any new instructions or tasks

### CODE REVIEW

**Comment by Andy-Millar** [comment]
Screenshot 2021-02-11 at 09 22 54
We've got it formatted in this way - would be great if we can tidy it to match - otherwise great!

**Comment by Andy-Millar** [comment]
Should also say 'handed over' in the printout view

File: common/presenters/message-banner/assessment-to-handed-over-banner.js [hunk 1]
```diff
@@ -33,6 +33,17 @@ module.exports = function assessmentToHandedOverBanner({
           context,
         })}
       </a>
+
+      <p class="app-icon app-print--hide">
+        ${i18n.t('messages::assessment.handed_over.note', {
+          context,
```
Comment (teneightfive) [inline common/presenters/message-banner/assessment-to-handed-over-banner.js:39]: You can skip passing `context` and `details` in as they're not used by the translation.

File: common/presenters/message-banner/assessment-to-handed-over-banner.js [hunk 2]
```diff
@@ -33,6 +33,17 @@ module.exports = function assessmentToHandedOverBanner({
           context,
         })}
       </a>
+
+      <p class="app-icon app-print--hide">
```
Comment (teneightfive) [inline common/presenters/message-banner/assessment-to-handed-over-banner.js:37]: You can copy the markup from [this change](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1321/files#diff-07a3ade12b6c6f2871d31e04cb06e5f4c9c1ea66bbd06259216c803224d43500R57) to add this I think. In fact, we could just bundle into that change...

**Comment by teneightfive** [comment]
Added into #1321

**Pull request closed without merging**